graphql_compiler/schema_generation/schema_graph.py

```diff
@@ -364,10 +364,16 @@ def __repr__(self):
 
 
 class VertexType(SchemaElement):
+
+    # The parent class' __init__ method is marked abstract, and must be overridden.
+    # See the parent class for more details.
+    # pylint: disable=useless-super-delegation
     def __init__(self, class_name, abstract, properties, class_fields):
         """See base class __init__ method."""
         super(VertexType, self).__init__(class_name, abstract, properties, class_fields)
 
+    # pylint: enable=useless-super-delegation
```

Comment:
I think that these can actually be removed. Only the methods marked `@abstractmethod` actually have to be overriden.

Reply:
I thought the same thing at first. See lines 368-369 above.
```
    # The parent class' __init__ method is marked abstract, and must be overridden.
    # See the parent class for more details.
```